lib/file.txt

```diff
@@ -0,0 +1 @@
+{"name":"hello"}
```

Comment:
This file was created only for testing purpose right?

Reply:
Good catch I'll remove this.